Simulator/Core/Driver.cpp

```diff
@@ -68,6 +71,16 @@ void serializeSynapses();
 ///  @param  argv    arguments.
 ///  @return -1 if error, else if success.
 int main(int argc, char *argv[]) {
+
+   // Doesn't work if placed in parseCommandLine because -c is a required flag,
+   // and ParamContainer throws an error
+   if (argc == 2) {
+      if (strcmp(argv[1], "-v") == 0 || strcmp(argv[1], "--version") == 0){
+         cout << "Git commit ID: " << GIT_COMMIT_ID << endl;
+         exit(0);
+      }
+   }
+
```

Comment:
Let's fix this in our usage of ParamContainer, rather than adding separate code just for this.

Reply:
If it executes in `parseCommandLine`, then this gets printed to the terminal first:

```[TRACE][2021-07-22 22:28:43] Instantiating Simulator
[TRACE][2021-07-22 22:28:43] Parsing command line```